MAUI/Calendar/events.md

```diff
@@ -179,3 +179,236 @@ private void OnCalendarLongPressed(object sender, CalendarLongPressedEventArgs e
 {% endhighlight %}
 {% endtabs %}
 
+## Commands
+
+### ViewChangedCommand
+
+SfCalendar includes a built-in event called `ViewChanged` and invokes this event through the [ViewChangedCommand]() whenever the view is navigated to either the previous or next view. Additionally, the event is triggered when switching between different calendar views, with the CalendarViewChangedEventArgs provided as a parameter.
+
```

Comment:
The `SfCalendar` includes a built-in event called `ViewChanged`, which is triggered whenever the calendar view is navigated to either the previous or next view. This event can be invoked through the [ViewChangedCommand](), and it also triggers when switching between different calendar views. The `CalendarViewChangedEventArgs` is provided as a parameter to this event.

Reply:
Content updated.